components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -403,7 +401,7 @@ const ContributorProfileTab = ({
               </div>
             </div>
             <div className="mt-2 h-36">
-              <CardLineChart lineChartOption={chart} className="!h-36" />
+              <CardLineChart contributor={githubName} repoIds={[]} className="!h-36" />
```

Comment:
refactor (nit): Any reason the `repoIds` is always an empty array? If the field is optional, consider adding a default value if the prop isn't required.